lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -427,6 +434,40 @@ def test_enroll_member_student(self):
             CourseEnrollment.objects.filter(course_id=self.course.id, user=student).exists()
         )
 
+    def test_ccx_invite_enroll_up_to_limit(self):
+        """
+        enrolls a list of students up to the enrollment limit
+        """
+        self.make_coach()
+        # create ccx and limit the maximum amount of students that can be enrolled to 2
+        ccx = self.make_ccx(max_students_allowed=2)
+        ccx_course_key = CCXLocator.from_course_locator(self.course.id, ccx.id)
+        # create some users
+        students = [
+            UserFactory.create(username="test{0}".format(seq), is_staff=False) for seq in range(3)
```

Comment:
Do you need this code some where ```username="test{0}".format(seq)```. UserFactory manages its sequence of creation of multiple users. Please see below code:
https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/tests/factories.py#L76
https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/tests/factories.py#L77